src/main/java/net/minecraftforge/registries/RegistryBuilder.java

```diff
@@ -230,6 +244,57 @@ public RegistryBuilder<T> hasTags()
         return this;
     }
 
+    /**
+     * <p>Register this registry as an unsynced datapack registry, which will cause data to be loaded from
+     * a datapack folder based on the registry's name. The mod that registers this registry does not need to exist
+     * on the client to connect to servers with the mod/registry.</p>
+     * <p>The registry's name must be of the form {@code modid:modid/folder}.
+     * Data JSONs will be loaded from {@code data/<datapack_namespace>/modid/folder/}, where modid is the mod that registered this registry.</p>
+     * 
+     * @param codec the codec to be used for loading data from datapacks on servers
+     * @return this builder
+     * 
+     * @see {@link #dataPackRegistry(Codec, Codec)} for registering a synced datapack registry
```

Comment:
According to the documentation for the `@see` tag[^1], when the first character is neither a `"` (for standard text entry) nor `<` (for an external HTML link), then `javadoc` expects the text immediately following `@see` to be a class member reference. In line with this, the use of `{@link}` should be removed.

Furthermore, javadocs take the text after the reference in a `@see` tag as the label for the link[^2], akin to how `<a href="bleh.html">boop</a>` shows "boop" for the link. In line with this, the label should also be removed.

```suggestion
     * @see #dataPackRegistry(Codec, Codec)
```

If you wish to attach prose explanation when linking to the method, you'll need to add it as sa paragraph, though I would think the `@see` tag would be visible enough.

[^1]: [Java 17](https://docs.oracle.com/en/java/javase/17/docs/specs/javadoc/doc-comment-spec.html#see) or [Java 7 (with examples)](https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#see) documentation on the `@see` tag
[^2]: `@see` custom labels are used to change the label from the label auto-generated from the reference.[^1]

Reply:
Changed in next push

Reply:
Changed in next push